src/charclass.cpp

```diff
@@ -27,19 +27,35 @@
 
 void initClass(int player)
 {
+    // TODO: Dynamic class list loading/saving from/to file
+    enum class classes_t
```

Comment:
Nnnnot exactly. The problem isn't so much what the values are as much as the casting and extra typing it introduces.

Regardless, it's good you brought this up. Magic numbers are A Bad Thing and should be eliminated.